unity-client/Assets/Scripts/MainScripts/DCL/Controllers/MapRenderer/MinimapMetadata.cs

```diff
@@ -92,10 +92,14 @@ public void AddSceneInfo(MinimapSceneInfo sceneInfo)
     /// <param name="userInfo">User info model</param>
     public void AddOrUpdateUserInfo(MinimapUserInfo userInfo)
     {
-        if (usersInfo.TryGetValue(userInfo.userId, out MinimapUserInfo existingUserInfo))
-            existingUserInfo = userInfo;
-        else
+        if (!usersInfo.ContainsKey(userInfo.userId))
```

Comment:
In general, unless there's a good reason for it, I prefer avoiding negated conditionals. Can we invert it?